cirq/ops/pauli_string.py

```diff
@@ -271,6 +298,50 @@ def merge_and_kickback(qubit: raw_types.Qid,
                                                    pauli_map.get(qubit0),
                                                    gate.pauli0,
                                                    gate.invert0)
-        assert quarter_kickback % 2 == 0, ('Impossible condition.  '
+        assert quarter_kickback % 2 == 0, (
+            'Impossible condition.  '
             'quarter_kickback is either incremented twice or never.')
         return quarter_kickback % 4 == 2
+
+
+# Ignoring type because mypy believes `with_qubits` methods are incompatible.
+class SingleQubitPauliStringGateOperation(  # type: ignore
```

Comment:
Will there be a `MultipleQubitPauliStringGateOperation` (A `Moment` where paulis are being applied to a set of qubits) or something of that nature in the future?

Reply:
I don't think so. The closest is `PauliString` itself.